tests/test_buffer.py

```diff
@@ -634,7 +688,8 @@ def test_SimpleArray_casting(self):
 
         # check if arrayplex can cast to simplearray
         self.assertEqual(
-            helper.test_load_arrayfloat64_from_arrayplex(arrayplex_float64), True)  # noqa: E501
+            helper.test_load_arrayfloat64_from_arrayplex(arrayplex_float64),
```

Comment:
Linter made me do it.  No change of logic in this function.

Reply:
Seems like there is some flexibility in flake8. When I use different platforms, the format is sometimes changed due to the linter as well.

Reply:
It could be configurable.  Reformatting the small piece in the function is fine.  We can fix the configuration when the linter is causing problems.